web/client-api/src/main/java/io/deephaven/web/client/api/JsTable.java

```diff
@@ -365,9 +362,9 @@ public String[] getAttributes() {
 
     /**
      * @param attributeName
-     * @return null if no property exists, a string if it is an easily serializable property, or a `Promise
-     *         <Table>
-     *         ` that will either resolve with a table or error out if the object can't be passed to JS.
+     * @return null if no property exists, a string if it is an easily serializable property, or a
+     *         {@code Promise<Table>} that will either resolve with a table or error out if the object can't be passed
```

Comment:
This is still wrong, I cant find a way so far that is valid javadoc (i.e. "spotless doesn't mangle it") and is valid TS ("my IDE actually shows me something readable").

It is possible we will need to push some of this into jsinterop-ts-defs, process javadoc contents somewhat.